packages/web-runtime/src/composables/head/useHead.ts

```diff
@@ -22,7 +22,9 @@ export const useHead = () => {
               .join(', ')
           }
         ],
-        ...(unref(favicon) && { link: [{ rel: 'icon', href: unref(favicon) }] })
+        ...(unref(favicon) && {
+          link: [{ rel: 'icon', type: 'image/svg+xml', href: unref(favicon) }]
```

Comment:
since `unref(favicon)` is the url of the favicon... could you please only use the `type: 'image/svg+xml'` if the url ends with `.svg`? 🤔

Reply:
Good point 👍

Reply:
quadro-checked, not really necessary with modern browsers, closing that pr